target_game.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun May 10 21:57:08 2020
+
+@author: Aly
+"""
+# Firstly I import packages that I might need
+
+from random import randint
+
+from psychopy.visual import Window, ImageStim
+from psychopy.event import getKeys, Mouse
+from psychopy.clock import Clock
+
+# I create a function which randomly changes the position of my target
+# I chose numbers depending on my DISP size (I want the target to be always presented inside the screen)
+# then the new position is set to the target, so the target can be moved
+
+def move_target_at_random_pos(target):
+    pos_x = randint(-700, 700)
+    pos_y = randint(-400, 400)
+    
+    new_pos = (pos_x, pos_y)
+    target.setPos(new_pos)

```

Comment:
Maybe this is a stupid comment but I don't see where you import setPos()

Reply:
I think you can ignore this comment. I'm a bit confused about what functions and methods etc are in python. Sorry!

Reply:
not a problem :) I use setPos to change the position of my target iage to the new position (that was randomly allocated through the randint function) :)